rpcs3/CMakeLists.txt

```diff
@@ -58,6 +58,10 @@ else()
 	set(ADDITIONAL_LIBS "")
 endif()
 
+if(UNIX)
```

Comment:
@danilaml This is not 100% true, there is a commercial project called moltenVK which allows your vulkan code to run on Metal. I've been considering trying to port RPCS3 to mac using this, so I would appreciate these left in.